app/views/schools/estimate_participants/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: root_path) # TODO: get back path
+%>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-3">How many early career teachers and mentors do you expect to have in your programme this year?</h1>
+    <p class="govuk-body">These can be rough estimates.</p>
+
+    <%= render 'form' %>
```

Comment:
There is only one _form within the estimate_participants folder, so we are unlikely to get confused ?

Reply:
Depends what we are doing. If we are searching for all uses of a particular partial (because we removed a field or two and want them inlined), I would hate all of our forms to live in files named `form`.

And I think in general we will have some folders that will contain more than one form - especially multi page flows - so it's unlikely naming things as `_form` will stick as a convention.